inference-engine/tests/functional/plugin/shared/include/behavior/infer_request_cancellation.hpp

```diff
@@ -41,31 +41,13 @@ TEST_P(CancellationTests, canCancelAsyncRequest) {
     auto execNet = ie->LoadNetwork(cnnNet, targetDevice, configuration);
     // Create InferRequest
     InferenceEngine::InferRequest req = execNet.CreateInferRequest();
-    bool cancelled = false;
```

Comment:
do we have test that not started infer can be cancelled / does nothing?

Reply:
`canCancelBeforeAsyncRequest`